sky/shell/shell.cc

```diff
@@ -108,6 +137,7 @@ Shell::Shell() {
   blink::SetServiceIsolateHook(ServiceIsolateHook);
   blink::SetRegisterNativeServiceProtocolExtensionHook(
       PlatformViewServiceProtocol::RegisterHook);
+  blink::SetLookupFileNameForSymbolNameHook(LookupFileNameForSymbolNameHook);
```

Comment:
Is there some advantage to using a callback?  It seems disadvantageous for two reasons:

1) The code is less direct.  (Literally, there's an indirect jump that takes some puzzling to track down.)
2) The code is more complicated because the callback loops over the files to find the right one instead of just setting each property in order.